Sources/Player/Publishers/AVPlayerItemPublishers.swift

```diff
@@ -36,6 +36,21 @@ extension AVPlayerItem {
         .eraseToAnyPublisher()
     }
 
+    func loadedTimeRangePublisher() -> AnyPublisher<CMTimeRange, Never> {
```

Comment:
From an API consistency perspective with other publishers like `timeRangePublisher()` maybe we should return `.invalid` when the range is unknown?